front/app/containers/App/index.tsx

```diff
@@ -239,6 +239,26 @@ class App extends PureComponent<Props, State> {
               },
             });
           });
+        } else if (
+          tenant.data.attributes.style &&
+          tenant.data.attributes.style.customFontURL
+        ) {
+          import('webfontloader').then((WebfontLoader) => {
+            const fontName = (
+              tenant.data.attributes.style as IAppConfigurationStyle
+            ).customFontName;
+            const fontURL = (
+              tenant.data.attributes.style as IAppConfigurationStyle
```

Comment:
A couple of suggestions: we should use `as` as little as possible. I know these lines are based on the code above, but it's bad practice there. `as` should be used as a last resort only. This would work just fine:

const fontName = tenant.data.attributes.style?.customFontName;
const fontURL = tenant.data.attributes.style?.customFontURL;

Or to make it even better:

```
const styleSettings = tenant.data.attributes.style;

if (styleSettings) {
  const fontName = styleSettings.customFontName;
  const fontURL = styleSettings.customFontURL;

  if (fontName !== undefined && fontURL !== undefined) {
    import('webfontloader').then((WebfontLoader) => {
      WebfontLoader.load({
        custom: {
          families: [fontName],
          urls: [fontURL],
        },
      });
    });
  }
}
```

This will only import webfontloader when there's a custom font. Now it always imports it. The same correction could be made to the Adobe font loading.

And then a minor code readability remark: I'd not add it in an `if else` because it's not really tied to the Adobe font (but if else gives that impression when you're first reading the code).

@amanda-anderson everything you have discussed in the week planning has priority, but in case you have some spare time, feel free to open a new PR to make the suggested changes if you agree. Happy to review. :-)

Reply:
Thanks for the feedback! I'll make some updates later this week.